src/clj/mmmanyfold/routes/owlet.clj

```diff
@@ -1,16 +1,22 @@
 (ns mmmanyfold.routes.owlet
-  (:require [compojure.core :refer [defroutes GET]]
-            [ring.util.http-response :refer [ok not-found]]
+  (:require [compojure.core :refer [defroutes GET POST PUT]]
+            [ring.util.http-response :refer [ok not-found internal-server-error]]
             [compojure.api.sweet :refer [context]]
             [org.httpkit.client :as http]
+            [mailgun.mail :as mail]
             [cheshire.core :as json]))
 
+(def creds {:key    (System/getenv "PLAYGROUND_MAILGUN_API_KEY")
```

Comment:
the api keys aren't tied to any specific email address so i think it's fine to use with any domain on this particular account